slurmstepd: info: Setting TMPDIR to /scratch/10912152. Previous errors about TMPDIR can be discarded
Slurm job started at 08/03/2025_11:11:02 AM

In transit.
Initing Ray (1 head node + 8 worker nodes + 8 simulation nodes) on nodes: 
	ilk-216 ilk-237 ilk-240 ilk-241 ilk-242 ilk-243 ilk-244 ilk-245 ilk-246 ilk-247 ilk-248 ilk-249 ilk-250 ilk-251 ilk-252 ilk-253 ilk-254
Head node: 172.30.205.24:6379


RAY_DEPLOY_TIME:     24.17751
2025-03-08 11:11:30,164	INFO packaging.py:520 -- Creating a file package for local directory '/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E2/DASK_ON_REISA/reduction/P16-SN8-LS536-GS8589-I20-AN8-D2025-03-08_09-07-52'.
2025-03-08 11:11:30,191	INFO packaging.py:347 -- Pushing file package 'gcs://_ray_pkg_f821e7c3c2ddc29e.zip' (0.48MiB) to Ray cluster...
2025-03-08 11:11:30,197	INFO packaging.py:360 -- Successfully pushed file package 'gcs://_ray_pkg_f821e7c3c2ddc29e.zip'.
RAY_INIT_TIME:       5.06129
Iter [0]
[2m[36m(iter_task pid=1981578, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981578, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981579, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981579, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981577, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981577, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981576, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981576, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981580, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981580, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981581, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981581, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=1981575, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981575, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=507916, ip=10.120.211.14)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=507916, ip=10.120.211.14)[0m   warnings.warn(
[2m[36m(iter_task pid=507919, ip=10.120.211.14)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=507919, ip=10.120.211.14)[0m   warnings.warn(
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m   warnings.warn(
[2m[36m(iter_task pid=507918, ip=10.120.211.14)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=507918, ip=10.120.211.14)[0m   warnings.warn(
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m   warnings.warn(
[2m[36m(iter_task pid=1981957, ip=10.120.205.45)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=1981957, ip=10.120.205.45)[0m   warnings.warn(
[2m[36m(iter_task pid=3019848, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019848, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019849, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019849, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019852, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019852, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019851, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019851, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019850, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019850, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019853, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019853, ip=10.120.211.19)[0m   warnings.warn(
[2m[36m(iter_task pid=3019854, ip=10.120.211.19)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3019854, ip=10.120.211.19)[0m   warnings.warn(
Iter [2]
Iter [4]
Iter [6]
Iter [8]
Iter [10]
Iter [12]
Iter [14]
Iter [16]
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m 2025-03-08 11:18:20,895	WARNING worker.py:2519 -- Local object store memory usage:
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m 
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m (global lru) capacity: 72000000000
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m (global lru) used: 17.8957%
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m (global lru) num objects: 2
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m (global lru) num evictions: 172
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m (global lru) bytes evicted: 188184965492
[2m[36m(iter_task pid=507920, ip=10.120.211.14)[0m 
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m 2025-03-08 11:18:38,260	WARNING worker.py:2519 -- Local object store memory usage:
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m 
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m (global lru) capacity: 72000000000
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m (global lru) used: 29.8262%
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m (global lru) num objects: 3
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m (global lru) num evictions: 172
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m (global lru) bytes evicted: 188184965492
[2m[36m(iter_task pid=507917, ip=10.120.211.14)[0m 
The node with node id: 43c8ba317a65401e94e97738a4b4d15fa48db87146c8dbd2aefc2185 and address: 172.30.211.20 and node name: 172.30.211.20 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a 	(1) raylet crashes unexpectedly (OOM, preempted node, etc.) 
	(2) raylet has lagging heartbeats due to slow network or busy workload.
EST_ANALYTICS_TIME:  433.12915 (avg:21.65646)
Traceback (most recent call last):
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E2/DASK_ON_REISA/reduction/P16-SN8-LS536-GS8589-I20-AN8-D2025-03-08_09-07-52/reduction.py", line 56, in <module>
    result = handler.get_result(process_func, iter_func, selected_iters=iterations, kept_iters=max_iterations,
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E2/DASK_ON_REISA/reduction/P16-SN8-LS536-GS8589-I20-AN8-D2025-03-08_09-07-52/reisa.py", line 127, in get_result
    tmp = ray.get(results)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 104, in wrapper
    return getattr(ray, func.__name__)(*args, **kwargs)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/api.py", line 42, in get
    return self.worker.get(vals, timeout=timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/worker.py", line 434, in get
    res = self._get(to_get, op_timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/worker.py", line 462, in _get
    raise err
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/server/server.py", line 489, in _get_object
    items = ray.get(objectrefs, timeout=request.timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/worker.py", line 2521, in get
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError: [36mray::iter_task()[39m (pid=507917, ip=172.30.211.14)
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E2/DASK_ON_REISA/reduction/P16-SN8-LS536-GS8589-I20-AN8-D2025-03-08_09-07-52/reisa.py", line 116, in iter_task
    current_results_list = ray.get(current_results_list)  # type: #List[dask.array.core.Array]
ray.exceptions.OwnerDiedError: Failed to retrieve object f1d34811cc0e962bffffffffffffffffffffffff0100000001000000. To see information about where this ObjectRef was created in Python, set the environment variable RAY_record_ref_creation_sites=1 during `ray start` and `ray.init()`.

The object's owner has exited. This is the Python worker that first created the ObjectRef via `.remote()` or `ray.put()`. Check cluster logs (`/tmp/ray/session_latest/logs/*d40f14fef7ab9f1ada0ca98916d47607711f3f2131d396740ac2d7fe*` at IP address 172.30.211.24) for more information about the Python worker failure.
srun: error: ilk-237: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=10912152.18
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** JOB 10912152 ON ilk-216 CANCELLED AT 2025-03-08T11:31:17 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 10912152.19 ON ilk-247 CANCELLED AT 2025-03-08T11:31:17 DUE TO TIME LIMIT ***
*** SIGTERM received at time=1741429877 on cpu 15 ***
*** SIGTERM received at time=1741429877 on cpu 62 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
*** SIGTERM received at time=1741429877 on cpu 15 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
*** SIGTERM received at time=1741429877 on cpu 16 ***
[failure_signal_handler.cc : 329] RAW: Signal 15 raised at PC=0x14af2407354c while already in AbslFailureSignalHandler()
*** SIGTERM received at time=1741429877 on cpu 15 ***
[failure_signal_handler.cc : 329] RAW: Signal 15 raised at PC=0x153e49cdf54c while already in AbslFailureSignalHandler()
*** SIGTERM received at time=1741429877 on cpu 62 ***
PC: @     0x14af2407354c  (unknown)  __read
    @     0x14af23fb79c0       3456  (unknown)
PC: @     0x153e49cdf54c  (unknown)  __read
    @     0x153e49c239c0       3456  (unknown)
